openpype/plugins/publish/collect_input_representations_to_versions.py

```diff
@@ -22,26 +22,30 @@ def process(self, context):
         # Query all version ids for representation ids from the database once
         representations = set()
         for instance in context:
-            inst_repre = instance.data.get("inputRepresentations", [])
-            representations.update(inst_repre)
+            inst_repre = instance.data.get("inputRepresentations")
+            if inst_repre:
+                representations.update(inst_repre)
```

Comment:
Anyone else prefers the old `.get()` style just because it hints what type we're expecting here?